src/components/setpieces/TeamMembers.astro

```diff
@@ -1,12 +1,18 @@
 ---
 import TeamMember from '@parts/team/TeamMember.astro';
 import teamData from '@data/Team';
+import teamInfo from "cache/teamInfo.json";
```

Comment:
I think Astro is able to directly fetch JSON data from an arbitrary URL, actually — see https://docs.astro.build/en/guides/data-fetching/ — so this would probably just work:
```js
const teamInfo = await fetch("https://team-info.quiltmc.org/").then((r) => r.json());
```
Though I have no idea whether that would incur any significant performance/speed penalties

Reply:
The worker takes a few seconds to fetch the data, I'd rather only do that once in dev